packages/seo-integration/tests/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from "enzyme";
+import EnzymeAdapter from "enzyme-adapter-react-16";
```

Comment:
Should we consider adding these imports to the `package.json` file for `seo-integration`?

Reply:
Yes, that is a good idea!

Since we only use it in the tests, we should add these as a dev dependency. I would also say that we should use the same versions as in `package.json` of the repository, to avoid version conflicts, so `^3.11.0` for `enzyme` and `^1.15.6` for `enzyme-adapter-react-16`.